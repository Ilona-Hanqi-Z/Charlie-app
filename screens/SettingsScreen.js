import React from 'react';
import { Text, View, Button } from 'react-native';

function SettingsScreen({navigation}) {
    return (
      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
        <Text>Settings Screen</Text>
       
      </View>
    );
}

export default SettingsScreen;