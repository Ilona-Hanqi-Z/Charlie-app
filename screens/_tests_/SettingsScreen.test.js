import React from 'react';
import { render } from 'react-native-testing-library';
import SettingsScreen from '../SettingsScreen';

test('it renders all inputs as expected', ()=> {
    const { toJSON } = render(<SettingsScreen />);

    expect(toJSON()).toMatchSnapshot();
})