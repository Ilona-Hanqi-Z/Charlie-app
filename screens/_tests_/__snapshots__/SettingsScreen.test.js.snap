// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders all inputs as expected 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Settings Screen
  </Text>
  <View
    accessibilityRole="button"
    accessibilityState={Object {}}
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "margin": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Click Here
      </Text>
    </View>
  </View>
</View>
`;
